po/de.po

```diff
@@ -178,7 +178,7 @@ msgstr "Token einem Benutzer zuweisen"
 
 #: privacyidea/static/components/token/views/token.getserial.html:33
 msgid "Assignment State"
-msgstr "Zuweisung"
+msgstr "Zuweisungsstatus"
```

Comment:
I personally do not like these long concatenated german words. User "Status der Zuweisung"